client/components/app.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    //state
+    this.state = {
+
+    }
+    //function bindings
+  }
+  render() {
+    return (
+    <div>PHOTOS</div>
+    )
+  }
+
+}
+
+export default App;
```

Comment:
@dekim2324 All of these files have a newline warning signs. I think that's a styling/common practice thing.